### DISCUSSION TITLE
Radio Buttons

### DISCUSSION THREAD

**Discussion by insomniacj93** [Budibase/budibase#7012] [Help] [answered]
Hello,

While using the Option Picker (Radio Buttons), The application doesn't seem to be getting the value from the radio button selected.
I've tried updating the form value but doesnt't work.
Is there any specific way to store the radio button value?

![print8](https://user-images.githubusercontent.com/107686004/182096897-c508885f-078d-48ae-a35e-1a05963f071e.png)

Thanks in advance!

**Comment by melohagan** [comment, marked as answer]
Hello @insomniacj93 

Could you provide some reproduction steps please? I can't immediately find any issue.

It looks like a validation error. So the 'origin' field requires a value to be provided. Is the Options Picker field named as 'origin'? 
Is your **Save Row** action saving the correct form?

**Reply by insomniacj93** [reply]
Hi @melohagan, I've found the solution in step forms.
Basicly I had 2 forms (one inside the other) and when saving, it would not recognise the value given to the columns in the second table, by providing the step form and removing one of the forms I was able to store the information in both tables without problem.

Sorry I forgot to upload it!
